BlockEQ.xcodeproj/project.pbxproj

```diff
@@ -1384,7 +1384,7 @@
 			inputFileListPaths = (
 			);
 			inputPaths = (
-				"${PODS_ROOT}/Target Support Files/Pods-BlockEQ/Pods-BlockEQ-frameworks.sh",
+				"${SRCROOT}/Pods/Target Support Files/Pods-BlockEQ/Pods-BlockEQ-frameworks.sh",
```

Comment:
@ndizazzo May I delete this and use your PATHS?

Reply:
Yeah - it's weird that this would change. No reason to commit it along with the PR.

Does your project build with them being the old format: `${PODS_ROOT}`? 

I recently updated our podfile to specify cocoapods version 1.6.0 beta 2 format, so I'm wondering if the format for environment variables changed between the last release and the one we're using now...